ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -123,31 +73,6 @@
         - "{{ role_path }}/tasks/partials/tap2junit/pip.yml"
       skip: true
 
-- name: Firewall | add rule to allow accepting multicast
-  lineinfile:
-    dest: /etc/sysconfig/iptables
-    insertafter: ":OUTPUT ACCEPT.*]"
-    line: "-A INPUT -m pkttype --pkt-type multicast -j ACCEPT"
-  when: "'rhel72-s390x' in inventory_hostname"
-
-- name: Firewall | add basic rule to allow communication locally
-  lineinfile:
-    dest: /etc/sysconfig/iptables
-    insertafter: ":OUTPUT ACCEPT.*]"
-    line: "-A INPUT -s 127.0.0.1/32 -d 127.0.0.1/32 -j ACCEPT"
-  when: "'rhel72-s390x' in inventory_hostname"
-
-- name: Firewall | add additional rule to allow communication from 127.0.0.2
-  lineinfile:
-    dest: /etc/sysconfig/iptables
-    insertafter: ":OUTPUT ACCEPT.*]"
-    line: "-A INPUT -s 127.0.0.2/32 -d 127.0.0.1/32 -j ACCEPT"
-  when: "'rhel72-s390x' in inventory_hostname"
-
-- name: Firewall | make the new firewall rules take effect
-  command: systemctl restart iptables
-  when: "'rhel72-s390x' in inventory_hostname"
-
```

Comment:
Are these no longer required? Or moved to a manual step?

Reply:
No longer required I believe, These steps failed on the machines but make+test still pass. Need to try with a non master branch to double check

Reply:
@AshCripps @sam-github I think these rules were for libuv: https://github.com/nodejs/build/issues/2080#issuecomment-567280766 and https://github.com/libuv/libuv/pull/2185#issuecomment-467290846